index.html

```diff
@@ -0,0 +1,81 @@
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">
+
+</head>
+<body>
+    <div class="wrapper">
+      <div class="profile">
+        <div class="photo">
+          <img src="img/profile.jpg">
+        </div>
+        <div class="about">
+          <div class="name">
+            <h1>Lapitski Viachaslau</h1>
+            <h2>Front-end developer</h2>
+          </div>
+        </div>
+      </div>
+      <div class="content-field">
+        <div class="content-name">
+          <p>Summary:</p>
+        </div>
+        <div class="content">
+          <p>I'm an ambitious student of "The Rolling Scopes School". I'm a quick study, willing to be a part of 
+                        some great team as a front-end developer. I would like to work hard and improve new hard and soft skills.
+                        I can quickly manage with all planned tasks and cope with challenges as well, because big business deal
+                        with both and these skills are extremely necessary for all people involved in it. I can really assess
+                        my strengths and set deadlines.</p>
+        </div>
+        </div>
+        <div class="content-field">
+            <div class="content-name">
+                <p>Education:</p>
+            </div>
+            <div class="content">
+                <p>BSU Faculty of Mechanics and Mathematics, 2015-2020</p>
+                <p>New Front-End Adventures Workshop, September 2018 </p>
+                <p>English(Pre-intermediate(A2))</p>
+            </div>
+        </div>
+        <div class="content-field">
+            <div class="content-name">
+                <p>Skills:</p>
+            </div>
+            <div class="content">
+                <p>Git</p>
+                <p>JavaScript</p>
+                <p>HTML5</p>
+                <p>CSS3</p>
+            </div>
+        </div>
+        <div class="content-field">
+                <div class="content-name">
+                    <p>Experience:</p>
+                </div>
+                <div class="content">
+                    <p><a href="https://github.com/ogurez/sudoku">Sudoku Solver</a></p>
+                    <p><a href="https://github.com/ogurez/advanced-zeros">Finding trailing zeros</a></p>
+                    <p><a href="https://github.com/ogurez/brackets">Brackets sequence</a></p>
+                </div>
+            </div>
+        <div class="content-field">
+            <div class="content-name">
+              <p>Contacts: </p>
+            </div>
+            <div class="content">
+              <p> E-mail : <a href="mailto:ogurezslava@gmail.com?subject=Job">ogurezslava@gmail.com</a></p>
+              <p>Phone : <a href="tel:+375291937714">+375 29 193 77 14</a></p>
+              <p>Github : <a href="https://github.com/ogurez"><i class="fab fa-github"></i></a></p>
```

Comment:
Actually, we dont't need "i" here to plug the github logo. If you will take a look to the styles, you will see, that all the magic is just in css:
`element::after { content: "\f09b"; font-family: "Font Awesome 5 Brands"; }`
So you can just create a special pseudo element in css each time you need fa icon